src/tests.rs

```diff
@@ -359,3 +364,33 @@ async fn temp_alias_async() -> anyhow::Result<()> {
 
     Ok(())
 }
+
+#[tokio::test]
+async fn gc_loop() -> anyhow::Result<()> {
+    let store = BlockStore::memory(Config::default())?;
+    let store = AsyncBlockStore::new(TokioRuntime, store);
+    // let gc run in the background
+    let gc_loop = store
+        .clone()
+        .gc_loop(Duration::from_millis(100), 1000, Duration::from_secs(1));
+    let handle = tokio::spawn(gc_loop);
+
+    // add 2 blocks, one temp aliased, one not
+    let a = cid("a");
+    let b = cid("b");
+    let alias = store.temp_alias().await;
+    store
+        .add_block(a, b"fubar".to_vec(), vec![], Some(alias.clone()))
+        .await?;
+    store.add_block(b, b"fubar".to_vec(), vec![], None).await?;
+    // give GC opportunity to run
+    tokio::time::sleep(Duration::from_secs(1)).await;
```

Comment:
I guess you could tune that duration down quite a bit for the purpose of this test

Reply:
I don't want the test to be flaky. I experimented with the tokio time testing stuff, but it did not immediately work for me. But I can maybe reduce it a bit.